src/backend/marsha/core/models/video.py

```diff
@@ -554,3 +554,64 @@ class Meta:
         db_table = "device"
         verbose_name = _("device")
         verbose_name_plural = _("devices")
+
+
+class SharedLiveMedia(UploadableFileMixin, BaseModel):
+    """Model representing a shared media in a video live."""
+
+    RESOURCE_NAME = "sharedlivemedias"
+
+    video = models.ForeignKey(
+        Video,
+        on_delete=models.CASCADE,
+        related_name="sharedlivemedias",
+        verbose_name=_("Video"),
+    )
+
+    title = models.CharField(
+        blank=True,
+        null=True,
+        max_length=255,
+        verbose_name=_("title"),
+        help_text=_("title of the shared live media"),
+    )
+
+    show_download = models.BooleanField(default=True)
+
+    pages = models.PositiveIntegerField(
```

Comment:
Should be called nb_pages?

Reply:
or pages_count ?

Reply:
To be honest none of your propositions look better than mine 😅 when I read `pages` I expect a number in document context. Not you ?

Reply:
Not me because in richie there is a Page model so everything called `page` is actually a page.
nb_pages or pages_count are both better in my opinion because this field is a number or a count, not a page.

Reply:
Using the plural form is not confusing and we are not creating a CMS, the context is not the same. The context matters here, like in a LTI request 😂

Reply:
We disagree. Let's make a poll on slack? 🤷‍♂️

Reply:
No, it's not needed to waste time on this subject IMO, I will take one of these suggestions